src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,164 @@
+package duke;
+
+import java.util.List;
+
+import javafx.application.Application;
+import javafx.scene.layout.Region;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import duke.exceptions.DukeException;
+import duke.exceptions.UnknownInputException;
+import duke.tasks.DeadlineTask;
+import duke.tasks.EventTask;
+import duke.tasks.Task;
+import duke.tasks.ToDoTask;
```

Comment:
Same thing about import statements, consistent and readable. (coding standard)

Reply:
Same issue as above, I tried to separate the imports previously for better readability but it didn't manage to pass checkstyle